modules/apps/29-fee/fee_test.go

```diff
@@ -36,6 +36,14 @@ func (suite *FeeTestSuite) SetupTest() {
 	suite.path = path
 }
 
+func NewTransferPath(chainA, chainB *ibctesting.TestChain) *ibctesting.Path {
```

Comment:
Where are we using this?

Reply:
leftover from the integration test attempt